src/components/analytics/segmentProvider.tsx

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-unused-modules */
+import { AnalyticsBrowser } from '@segment/analytics-next'
+import React, { useMemo } from 'react'
+
+import { AnalyticsContext, AnalyticsContextProps } from './segmentContext'
+
+interface AnalyticsProviderProps {
+  children: React.ReactNode
+}
+
+export const AnalyticsProvider: React.FC<AnalyticsProviderProps> = ({ children }) => {
+  const writeKey = process.env.REACT_APP_SEGMENT_WRITE_KEY ?? ''
```

Comment:
Is `''` different than `null` or `undefined`?

Reply:
Just used the suggested way from segment docs